lib/run-status.js

```diff
@@ -166,6 +172,7 @@ RunStatus.prototype.processResults = function (results) {
 	this.tests = flatten(this.tests);
 
 	this.passCount = sum(this.stats, 'passCount');
+	this.knownFailureCount = sum(this.stats, 'knownFailureCount');
```

Comment:
Seems it is called in api.js, but since it has knownFailures, should I just use its length?